internal/faustjs.org/docs/getting-started.mdx

```diff
@@ -0,0 +1,87 @@
+---
+slug: getting-started
+title: Getting Started with Faust
+---
+
+# Getting Started with Faust
+
+## Quick Start
+
+To create a Faust project, run:
+
+```bash
+npx create-next-app \
+    -e https://github.com/wpengine/faustjs/tree/main \
+    --example-path examples/next/faustwp-getting-started \
+    --use-npm
+```
+
+Now, `cd` into your new app and copy the sample environment template:
+
+```bash
+cp .env.local.sample .env.local
+```
+
+Finally, run the dev server:
+
+```bash
+npm run dev
+```
+
+You can now visit [http://localhost:3000](http://localhost:3000) to see your new project.
+
+<p align="center">
+  <video
+    src="/docs/video/next/getting-started-next-example.mp4"
+    muted
+    autoPlay
+    playsInline
+    controls
+    loop
+  />
+</p>
+
+Currently, the posts and pages you see are coming from our WordPress site at [https://faustexample.wpengine.com](https://faustexample.wpengine.com). In the next step, we'll show you how to hook up your own WordPress site.
+
+## Connecting your WordPress site
+
+The example app above loads WordPress content from the demo site at [https://faustexample.wpengine.com](https://faustexample.wpengine.com).
+
+To point it to a different WordPress site, first, make sure you have installed and activated the necessary WordPress plugins:
+
+- [Faust](https://wordpress.org/plugins/faustwp/)
+- [WPGraphQL](https://wordpress.org/plugins/wp-graphql/)
+
+Once the necessary plugins have been installed, open the `.env.local` file you created earlier, it should look something like this:
+
+```bash
+# Your WordPress site URL
+NEXT_PUBLIC_WORDPRESS_URL=https://faustexample.wpengine.com
+
+# Plugin secret found in WordPress Settings->Headless
+# FAUSTWP_SECRET_KEY=YOUR_PLUGIN_SECRET
+```
+
+Update the `NEXT_PUBLIC_WORDPRESS_URL` value with your WordPress site URL (be sure to include `http://` or `https://`).
+
+Additionally, update the `FAUSTWP_SECRET_KEY` value with the secret key found in Settings → Faust in your WordPress admin area to support previews.
+
+<img
+  src="/docs/img/headless-admin-secret.png"
+  alt="The FaustWP WordPress admin interface with a red rectangle around the Secret Key field"
+/>
+
+## Check Your Permalinks
+
+Finally, ensure your permalink structure in `WP Admin -> Settings -> Permalinks` **is not** set to "plain":
+
+<img
+  src="/docs/img/permalinks.png"
+  alt="WordPress Admin Settings Page for Permalinks"
+/>
+
+Faust resolves templates similar to the Template Hierarchy in WordPress, but "plain" permalinks are not supported due to constraints in Next.js.
+
+## Start Editing
+
+With your project created and development environment started, edit `wp-templates/front-page.js` and see the updated results in your browser.
```

Comment:
We probably need to add a link to the project walkthrough here as well. Check the last paragraph of the deleted `getting-started.mdx` file

Reply:
Done! @theodesp